cob_behaviour/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cob_behaviour)
```

Comment:
I like the idea of a `cob_behaviour` package....but I think this should be in `cob_command_tools` rather than `cob_robots`?